.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.6
```

Comment:
Since both `beberlei/assert` and most importantly `goaop/framework` (https://github.com/goaop/framework/releases/tag/2.3.0) drop support for php 5.6, I think php-deal should follow also.

Reply:
Looking into the `composer.json` from goaop. There is still `>=5.6` in there. So this still runs with 5.6.

I would suggest keeping the 5.6 for now, since I already have a Version 1.0 for this lib as a PR and this will be merged soon. Then the support for every php version less than 7.2 will be dropped.

Reply:
Indeed, it mentions that it will drop the support from v2.

Reply:
I just opened an issue asking if this is correct.
Anyways, with 1.0 the support will drop for sure 😉